apps/showcase/src/app/sdk/sdk.template.html

```diff
@@ -33,7 +33,11 @@ <h2 id="sdk-example">Example</h2>
 
     <h2 id="rules-engine-install">How to use</h2>
     <div>
-      <o3r-copy-text-pres [wrap]="true" language="bash" text="yarn create @ama-sdk typescript <project-name> [--spec-path=./path/to/spec.yaml]"></o3r-copy-text-pres>
+      <markdown clipboard ngPreserveWhitespaces>
+        ```bash
+        yarn create &#64;ama-sdk typescript &lt;project-name&gt; [--spec-path=./path/to/spec.yaml]
```

Comment:
```suggestion
        npm create &#64;ama-sdk typescript &lt;project-name&gt; [--spec-path=./path/to/spec.yaml]
```
?

`yarn create` is officially supported only on yarn1